addons/im_livechat/static/tests/embed/unread_messages.test.js

```diff
@@ -42,9 +41,12 @@ test("new message from operator displays unread counter", async () => {
         livechat_channel_id: livechatChannelId,
         livechat_operator_id: serverState.partnerId,
     });
+    onRpc("/web/dataset/call_kw/ir.http/lazy_session_info", () => {
+        asyncStep("lazy_session_info");
+    });
```

Comment:
Couldn't we just remove `lazy_session_info` in steps it if this is there for strangeness of test environment but irrelevant in practice?

Note that `waitForSteps("init_livechat")` is there to await enough time to make sure simulated `mail/message/post` afterward is made without compromising some RPCs from Mitchell Admin like the `mark_as_read`. So this is unfortunate leaking technical stuffs, so the less needed the better!